src/storage/CommonUtils.cpp

```diff
@@ -114,8 +114,8 @@ bool CommonUtils::checkDataExpiredForTTL(const meta::SchemaProviderIf* schema,
     return false;
 }
 
-std::pair<bool, std::pair<int64_t, std::string>>
-    CommonUtils::ttlProps(const meta::SchemaProviderIf* schema) {
+std::tuple<bool, int64_t, std::string>
```

Comment:
It is more intuitive to use std::optional<std::tuple<int64_t, std::string>> here.